cumulus_library_data_metrics/q_valid_us_core_v4/q_valid_us_core_v4.jinja

```diff
@@ -21,5 +21,5 @@ SELECT
 FROM src_valid
 LEFT JOIN src_status
 ON src_valid.id = src_status.id
-WHERE NOT src_valid.valid
+WHERE src_valid.valid IS NULL OR NOT src_valid.valid
```

Comment:
Real change here (commenting actual logic changes, because the refactoring into a utility method in this PR is a little noisy)